test/tests/materials/fp_isolator/tests

```diff
@@ -0,0 +1,27 @@
+[Tests]
+  [./shear_P]
+    type = CSVDiff
+    input = fp_isolator_shear_P.i
+    csvdiff = fp_isolator_shear_P_out.csv
+  [../]
+  [./shear_V]
+    type = CSVDiff
+    input = fp_isolator_shear_V.i
+    csvdiff = fp_isolator_shear_V_out.csv
```

Comment:
This can be written as:

```
[./shear_V]
   type = CSVDiff
   input = fp_isolator_shear_P.i
   csvdiff = fp_isolator_shear_V_out.csv
   cli_args = 'Materials/elasticity/pressure_dependent=false Materials/elasticity/velocity_dependent=true Outputs/file_base=fp_isolator_shear_V_out'
  prereq=shear_P
[../]
```
`cli_args` contains all the parameters you want to change in the input file and `prereq` makes sure the tests are run sequentially to prevent file overwriting errors.

Reply:
Should I delete the input files itself? or just change the "tests" file as above? 
Also what does prereq = shear_P mean? How should I edit this if I want to make shear_P, shear_T, shear_V and shear_PTV in one file?

To make it simple, I can actually remove tests for individual boolean flags and just retain shear_PTV test?

Reply:
prereq = shear_P means that the test named shear_P is a prerequisite (will be run prior to) the current test. Yes, you can remove the input files and change the tests file according to @sveerara's suggestions. It's good practice to use those techniques and minimize the number of input files in the tests in general. That way if you want to make changes in the syntax in the future, you don't have to change a bunch of input files.